Add Gothic Coat and Gabber Tracksuit Clothing Item

ivomink opened the pull request:
# Description

This PR adds a little cute gothic coat and gabber tracksuit to the game. You can pick it in the loadouts section in the lobby, going to outer and/or uniform. It's going to be there.
Alternatively, you could find 2 of gothic coat and 2 of gabber tracksuit in the AutoDrobe.
Both cost one point in the loadout.

EDIT: Another addition, The Hiphop Clown Mask! Given to both mime and clown in the loadout, with the option to take 2 of them from the autodrobe!
This addition is purely cosmetic and adds no balance value whatsoever.

---


<img width="147" height="192" alt="image" src="https://github.com/user-attachments/assets/4497c43f-3b94-4d7c-a1ce-793e119e2566" />
<img width="184" height="197" alt="image" src="https://github.com/user-attachments/assets/1036a91f-3ced-44ba-839b-8e08564f16d4" />
<img width="336" height="358" alt="image" src="https://github.com/user-attachments/assets/370d1aa6-9027-4817-9562-d1355e81df79" />


# Todo:
- [x] Add gabber tracksuit to loadout.
- [x] move the gothic coat to the AutoDrobe/theatre vending machine.
- [x] add a third piece of clothing.
---
:cl:
- add: Added Gothic Coat, Gabber Tracksuit, and Hiphop Clown mask to Clown and Mime loadouts and the AutoDrobe vending machine.

ivomink committed:
Add Gothic Coat clothing item and related assets

- Introduced `ClothingOuterGothicCoat` in `clothesmate.yml` with a quantity of 2.
- Updated `outerwearGroup.yml` to include `LoadoutClothingGothicCoat`.
- Added details for `ClothingOuterGothicCoat` in `MiscOuterwear.yml`, including components and description.
- Modified `outerClothing.yml` to specify the new loadout category and cost.
- Added new PNG images for `equipped-OUTERCLOTHING` and `icon`.
- Created `meta.json` with versioning, licensing, and state definitions for the new item.

ivomink committed:
Add Gabber Tracksuit clothing item and assets

- Updated theater.yml to include the new clothing item
`ClothingUniformGabberTracksuit` with a quantity of 2.
- Added .rsi and prototype of said clothing item.

ivomink commented:
Don't merge this yet, i'm going to make a few more changes.

M3739 commented:
> Don't merge this yet, i'm going to make a few more changes.

I have converted this PR to a draft PR to reflect your declaration of it being a WIP.

Feel free to make it back into an open PR to communicate that it's ready to review.

ivomink committed:
- Moved gothic coat to theatre vend
- added gabber tracksuit to loadouts

ivomink committed:
Add Hiphop Clown Mask

- Resprited it with credit given in the meta.json file
- Gave it to both clown and mime in loadouts
- You can take it from the AutoDrobe vending machine.

ivomink commented:
Seems like it's done, feel free to review.

Mnemotechnician reviewed (changes requested):
Looks fine, but please move the new prototypes to the _Floof namespace.
Additionally, I updated your changelog to follow the standards (your old one had an invalid format)

ivomink commented:
Only the new prototypes regarding the clown/mime loadout and mask itself right?

Mnemotechnician commented:
Yes. To be clear, it doesn't matter which file you put a prototype in, so long as it's inside /Resources/Prototypes/, the game will load it.

ivomink commented:
Gotcha, will get into it.

ivomink committed:
moved to _Floof

ivomink committed:
Moved the texture and fixed the prototype to include the correct path

ivomink commented:
I FUCKED IT UP WITH THE LAST 2 COMMITS LOL

ivomink commented:
i'll redo it later, PR closed.
i think it's just not the time
